src/contracts/core/StrategyManager.sol

```diff
@@ -909,6 +888,31 @@ contract StrategyManager is
         strategyWhitelister = newStrategyWhitelister;
     }
 
+
+    /**
+     * @notice internal function for updating strategy manager's accounting of shares for the beacon chain ETH strategy
+        * @param sharesDelta is the change in podOwner's beaconChainETHStrategy shares
+     */
+    function _updateSharesToReflectBeaconChainETHBalance(address podOwner, uint256 beaconChainETHStrategyIndex, int256 sharesDelta) internal {
+        
+        if (sharesDelta < 0) {
+                IStrategy[] memory strategies = new IStrategy[](1);
+                strategies[0] = beaconChainETHStrategy;
+                uint256[] memory shareAmounts = new uint256[](1);
+                int256 shareAmount = sharesDelta >= 0 ? sharesDelta : -sharesDelta;
+                shareAmounts[0] = uint256(shareAmount);
```

Comment:
you're already checking that `sharesDelta < 0` -- should these lines just be combined into:
```
shareAmounts[0] = uint256(-sharesDelta);
```
?